### PR TITLE
emacs*: add German translation

### PR DESCRIPTION
**Pull request by flyck**
- [X] The page (if new), does not already exist in the repo.
- [X]  The page is in the correct platform directory (common/, linux/, etc.)
- [X]  The page has 8 or fewer examples.
- [X]  The PR title conforms to the recommended templates.
- [X]  The page follows the content guidelines.
- [X]  The page description includes a link to documentation or a homepage (if applicable).

Hello,

I translated the two emacs pages into German.

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Comment by sbrl** [comment]
Welcome to  tldr pages :slightly_smiling_face:

File: pages.de/common/emacs.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`Ctrl + X, Ctrl + S`
+
+- Tastenkombination zum Beenden von Emacs:
+
+`Ctrl + X, Ctrl + C`
+
+- Öffnen einer Datei in einer bestimmten Zeile:
+
+`emacs +{{line_number}} {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.de/common/emacs.md:24]: Please translate the arguments for get this easier for non-English speakers, like `line_number` and `path/to/file` in both pages.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]
Thanks @flyck, please **don't** push force to the PRs, it's harder to us to track the changes and suggestions.

**Pull request merged**